src/help/index.ts

```diff
@@ -50,7 +50,7 @@ export abstract class HelpBase extends HelpFormatter {
 }
 
 export class Help extends HelpBase {
-  protected CommandHelpClass: typeof CommandHelp = CommandHelp
+  public CommandHelpClass: typeof CommandHelp = CommandHelp
```

Comment:
Modified visibility to allow one to override this value

Reply:
What issues were you running into with overriding protected methods? That's something you should be able to do. 

I tried the following and typescript didn't complain about it:

```typescript
export class MyCommandHelp extends CommandHelp {
  protected sections(): Array<{header: string; generate: HelpSectionRenderer }> {
    const generate = (() => 'a string') as HelpSectionRenderer;
    return [{ header: 'my section', generate }];
  }
}

export class MyHelp extends Help {
  protected CommandHelpClass: typeof MyCommandHelp = MyCommandHelp
}
```